apps/govern/components/VeOlas/IncreaseLockModal/IncreaseAmount.tsx

```diff
@@ -0,0 +1,144 @@
+import { Button, Divider, Flex, Form } from 'antd';
+import { ethers } from 'ethers';
+import { useState } from 'react';
+
+import { notifyError, notifySuccess } from 'libs/util-functions/src';
+
+import { hasSufficientTokensRequest, updateIncreaseAmount } from 'common-util/functions';
+import { useFetchBalances } from 'hooks/useFetchBalances';
+
+import { ApproveOlasModal } from '../ApproveOlasModal';
+import { MaxButton } from '../MaxButton';
+import { OlasAmountInput } from '../OlasAmountInput';
+import { ProjectedVeOlas } from '../ProjectedVeOlas';
+import { useVeolasComponents } from '../useVeolasComponents';
+
+type IncreaseAmountProps = {
+  closeModal: () => void;
+};
+
+type FormValues = {
+  amount: number;
+};
+
+export const IncreaseAmount = ({ closeModal }: IncreaseAmountProps) => {
+  const [form] = Form.useForm();
+  const { account, lockedEnd, olasBalance, veOlasBalance, refetch } = useFetchBalances();
+  const { getLockedAmountComponent } = useVeolasComponents();
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [isApproveModalVisible, setIsApproveModalVisible] = useState(false);
+
+  const amountInEth = Form.useWatch('amount', form);
+
+  const onIncreaseAmount = async () => {
```

Comment:
minor, suggesting `handleIncreaseAmount` and `handleFinish` for ln 34, and ln 53